clock.lua

```diff
@@ -43,5 +43,8 @@ function onReceiveMIDI(message, connections)
     stop()
   elseif message[1] == MIDIMessageType.CLOCK and playing then
     advanceClock()
+  elseif stash_control_changes and isControlChange(message) and not messageIsNRPN(message) then
```

Comment:
`not messageIsNRPN(message)` will exclude NRPNs. We can't include these right now because

1. The stash doesn't support multi value control and playback (yet)
2. We need to exclude the mute NRPN messages. We never want to play those back.

Unfortunately the drum channels are all controlled with NRPN messages. This will limit these new controls to being used for the synth channels for now.